conn_oob.go

```diff
@@ -238,50 +236,19 @@ func (info *packetInfo) OOB() []byte {
 		// 	struct in_addr ipi_spec_dst; /* Local address */
 		// 	struct in_addr ipi_addr;     /* Header Destination address */
 		// };
-		msgLen := 12
-		if runtime.GOOS == "freebsd" {
-			msgLen = 4
-		}
-		cmsglen := cmsgLen(msgLen)
-		oob := make([]byte, cmsglen)
-		cmsg := (*syscall.Cmsghdr)(unsafe.Pointer(&oob[0]))
-		cmsg.Level = syscall.IPPROTO_TCP
-		cmsg.Type = msgTypeIPv4PKTINFO
```

Comment:
Can you delete this constant? Note it’s defined a couple of times, for the different build architectures.

Reply:
This one is still being used here: https://github.com/lucas-clemente/quic-go/blob/master/conn_oob.go#L174. I will delete it with my next PR that uses the `x` package also for reading OOB.